browser/brave_wallet/brave_wallet_service_unittest.cc

```diff
@@ -1348,6 +1348,20 @@ TEST_F(BraveWalletServiceUnitTest, SetAssetSpamStatus) {
   tokenWithInvalidChain->chain_id = "invalid_chain_id";
   SetAssetSpamStatus(tokenWithInvalidChain.Clone(), true, &success);
   EXPECT_FALSE(success);  // Should fail because of invalid chain_id
+
+  // Set the spam_status of a token not in user assets list
+  LOG(ERROR) << "BEFORE RELEVANT TEST";
```

Comment:
Don't log in test. If you want to see something when error, use `SCOPED_TRACE` but seems like this is just a leftover.

Reply:
Yes just leftover, removed.